src/corona_hakab_model/consts.py

```diff
@@ -302,27 +312,27 @@ class ImmuneTerminalState(ImmuneState, TerminalState):
             contagiousness=self.mild_condition_infection_ratio,
             test_willingness=self.mild_condition_test_willingness,
         )
-        need_close_medical_care = ImmuneStochasticState(
+        need_close_medical_care = ContagiousStochasticState(
             "NeedOfCloseMedicalCare",
+            contagiousness=self.need_close_medical_care_infection_ratio,
             test_willingness=self.need_close_medical_care_test_willingness,
-            detectable=True,
         )
 
-        need_icu = ImmuneStochasticState(
+        need_icu = ContagiousStochasticState(
             "NeedICU",
-            detectable=True,
+            contagiousness=self.need_icu_infection_ratio,
             test_willingness=self.need_icu_test_willingness
         )
 
-        improving_health = ImmuneStochasticState(
+        improving_health = ContagiousStochasticState(
             "ImprovingHealth",
-            detectable=True,
```

Comment:
What happened to the detectability value of each state?

Reply:
fixed